marine/Cargo.toml

```diff
@@ -11,16 +11,16 @@ name = "marine"
 path = "src/lib.rs"
 
 [dependencies]
-marine-core = { path = "../core", version = "0.15.1" }
+marine-core = { path = "../core", version = "0.16.0" }
 marine-module-interface = { path = "../crates/module-interface", version = "0.4.1" }
 marine-utils = { path = "../crates/utils", version = "0.4.0" }
 marine-rs-sdk-main = { version = "0.6.15", features = ["logger"] }
 marine-rs-sdk = { version = "0.6.15", features = ["logger"] }
 it-json-serde = { path = "../crates/it-json-serde", version = "0.3.1" }
 
-wasmer-runtime = { package = "wasmer-runtime-fl", version = "=0.17.1" }
+wasmer-runtime = { package = "wasmer-runtime-fl", version = "=0.17.1"}
 # dynamicfunc-fat-closures allows using state inside DynamicFunc
-wasmer-core = { package = "wasmer-runtime-core-fl", version = "=0.17.1", features = ["dynamicfunc-fat-closures"] }
+wasmer-core = { package = "wasmer-runtime-core-fl", version = "=0.17.1", features = ["dynamicfunc-fat-closures", "generate-debug-information", "wasm-debug"] }
```

Comment:
Why these features are needed?

Reply:
I tried using them to analyze memory usage before i found memory leak. Now they are not needed.